src/migration/1660640181400-UniqueSocialHandles.ts

```diff
@@ -0,0 +1,27 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class UniqueSocialHandles1660640181400 implements MigrationInterface {
+  name = 'UniqueSocialHandles1660640181400';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
```

Comment:
I see that we had already index on `username` and `twitter`, but I don't see where we get rid of the old indexes here

Reply:
Oh, good catch. The migration result is a bit mess, I had to pick the new ones manually.